android/kotlin/AmpliApp/README.md

```diff
@@ -0,0 +1,30 @@
+# Ampli Android SDK (Kotlin)
+An example app using the Ampli Android SDK
+
+# Usage
+
+### Setup the project
+You will need to do the following before running the app.
+1. Create new file `src/main/res/values/secrets.xml` with your Amplitude API key:
+```xml
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you please add this file explicitly to git as described here, then gitignore